### PR TITLE
refactor(linearprogress): apply 2020 color palette

### PR DESCRIPTION
**Pull request by jaketrent**
asdf

### CODE REVIEW

File: packages/linearprogress/src/css/index.js [hunk 1]
```diff
@@ -13,23 +16,23 @@ export default {
     borderRadius: '4px'
   },
   [`.psds-linearprogress__bg.psds-theme--${themeDefaultName}`]: {
-    backgroundColor: transparentize(0.8, core.colors.bone)
+    backgroundColor: 'rgba(255, 255, 255, 0.15)'
```
Comment (danethurber) [inline packages/linearprogress/src/css/index.js]: 🤷‍♂ - no utility usage eh?

Comment (jaketrent) [reply packages/linearprogress/src/css/index.js]: It's not background utility.  And it's not a border.  Brian and I talked about semantics and came to the conclusion of using a one-off.

Comment (EdwardIrby) [inline packages/linearprogress/src/css/index.js]: This should be `core.colorsBorder.lowOnDark`

File: packages/linearprogress/src/css/index.js [hunk 2]
```diff
@@ -13,23 +16,23 @@ export default {
...
   },
   [`.psds-linearprogress__bg.psds-theme--${themeNames.light}`]: {
-    backgroundColor: core.colors.gray01
+    backgroundColor: 'rgba(0, 0, 0, 0.15)'
```
Comment (EdwardIrby) [inline packages/linearprogress/src/css/index.js]: `core.colorsBorder.lowOnLight`

**Pull request merged**